rq/worker.py

```diff
@@ -699,7 +699,8 @@ def prepare_job_execution(self, job, heartbeat_ttl=None):
         """Performs misc bookkeeping like updating states prior to
         job execution.
         """
-        timeout = (job.timeout or 180) + 60
+        base_timeout = job.timeout or 180
```

Comment:
Sorry for the nitpick, but can we make this simpler to read by explicitly checking for `-1`?
```python
if job.timeout == -1:
    timeout = -1
else:
    timeout = job.timeout or 180
```

Reply:
I think that's good. Will update the pr